flags.go

```diff
@@ -33,6 +33,7 @@ func FlagSet(opt *Options) *flag.FlagSet {
 	for name, desc := range AvailableFormatters() {
 		descFormatOption += s(4) + "- " + colors.Yellow(name) + ": " + desc + "\n"
 	}
+	descFormatOption += s(4) + "- " + colors.Yellow("<format>") + ": name of your custom format" + "\n"
```

Comment:
godog does not compile package on `help` otherwise it would show the custom format in the list. probably later we will make it to compile the package, since after 1.5 release of go compilation was slower, now it is getting faster again, probably after go 1.8 this will show the custom formats in `godog -help`